canonical-web-and-design/jaas-dashboard
# Display truncated error messages in the model list

## Done

- Extract the model status parsing into a sharable util.
- Display truncated error messages in the model list.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- You should see a 'real' error message in the blocked models list.

## Details

Fixes: #110 
Fixes: #136 

## Screenshots

<img width="262" alt="Screen Shot 2019-11-07 at 1 33 43 PM" src="https://user-images.githubusercontent.com/532033/68421114-3b823300-0163-11ea-98f3-21598455fffc.png">
